modules/local/openms_cometadapter.nf

```diff
@@ -31,6 +31,10 @@ process OPENMS_COMETADAPTER {
                 -out ${prefix}.idXML \\
                 -database ${fasta} \\
```

Comment:
also nitpicking:
```suggestion
                -out ${prefix}.idXML \\
                -database $fasta \\
```
explanation: the above line requires the curly brackets because otherwise `.idXML` will be expected to be a map element of `prefix`, but `fasta` doesnt need the brackets. True for many modules.